styles/_queries.css

```diff
@@ -0,0 +1,96 @@
+@media screen and (max-width: 525px) {
+    nav {
+        display: flex;
+        width: 100%;
+    }
+    .nav-words {
+        display: flex;
+        color: white;
+        padding: 0 1rem;
+        font-size: 1rem;
+        width: 100%;
+        text-align: center;
+    }
+    h1 {
+        text-align: center;
+        padding: 0rem;
+        margin: 0;
+        font-size: 1rem;
+        width: 100%;
+    }
+    .bigger-hero {
+      padding: 0;
+    }
+    .hero {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        flex-wrap: wrap;
+    }
+    .project-row {
+        display: flex;
+        width: 100%;
+        font-size: 1.4rem;
+        flex-wrap: wrap;
+    }
+    .project-text {
+        flex-grow: 1;
+        width: 100%;
+    }
+    .project-frame {
+        flex-grow: 1;
+        width: 100%;
+        padding: 1rem;
+    }
+}
+@media screen and (max-width: 900px) and (min-width: 526px) {
+    nav {
+        display: flex;
+        width: 100%;
+    }
+    .nav-words {
+        display: flex;
+        color: white;
+        padding: 0 1rem;
+        font-size: 1rem;
+        width: 100%;
+        text-align: center;
+    }
+    h1 {
+        text-align: center;
+        padding: 0rem;
+        margin: 0;
+        font-size: 1rem;
+        width: 100%;
+    }
+    .bigger-hero {
+      padding: 0;
+    }
+    .hero {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        flex-wrap: wrap;
+    }
+    .project-row {
+        display: flex;
+        width: 100%;
+        font-size: 1.4rem;
+    }
+    .project-text {
+        width: 50%;
+    }
+    .project-frame {
+        width: 50%;
+        position: relative;
+        overflow: hidden;
+    }
+    .project-img {
```

Comment:
Move this positioning (including the frame `position` and `overflow`) to the regular style and add `object-fit: contain`